.github/workflows/ci.yml

```diff
@@ -277,7 +277,7 @@ jobs:
         # Comment out all xfails but the ones that have a run=False condition.
```

Comment:
In Crashing Test logs we want to run:
- xfails with `run=False`
 
so shouldn't the sed be run before the Crashing ones?

Reply:
I think the point is the inverse. -runxfail runs all xfail tests out of which some might crash. That’s what the first run would do: all crashing and all xfail. The second run will run all xfail and regular but no crashing tests. It is a bit suboptimal since we will run a lot more tests than we need but fine..

Reply:
The problem with this is we won't know if a previously xfailing test has now started passing or crashing. Or a previously passing test has now started crashing or xfailing.

Reply:
Fair point. I believe that I have addressed your comment.